### PR TITLE
feat: js-sdk create data pool

### PR DESCRIPTION
**Pull request by Astewart1510**
### Create Data Pool JS-SDK
This feature creates the transactions to create a data pool represented by a smart contract on the algorand blockchain network. Installed a testing framework called "Jest". 
This feature includes the create data pool method that includes the following transactions:
- Deploy smart contract teal code
- Fund smart contract
- A fake "setup" transaction mimicking the future action of an enclave
- Claim original creator contributor token 

### Test
Testing is done by creating fake accounts and signing the transactions without the imported signing functionality from the wallet. 
- setup local sandbox and run 
- run in terminal `npm run test-sdk -- tests/sdk.test.ts --verbose`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,38 +33,32 @@
     "@msgpack/msgpack": "^2.8.0",
     "@ngneat/until-destroy": "^9.2.3",
     "@ngrx/store": "^15.4.0",
-    "@stablelib/hkdf": "^1.0.1",
```
Comment (Astewart1510) [inline package.json:34]: @jongbonga Im confused with what happened with the package.json files during the merge which is probably resulting in the conflicts?

Comment (jongbonga) [reply package.json:34]: did you pull the latest changes from dev? where did you branch out from?

Comment (Astewart1510) [reply package.json:34]: yeah I did a git fetch and rebase origin/development .. but I think when I had to accept or complete the changes I might have made a mistake..

Comment (jongbonga) [reply package.json:34]: Most issues are on the package json. I will try and detangle it today

**Pull request closed without merging**